service/test/controllers/usermanagement/validator/ShadowUserMigrateReqValidatorTest.java

```diff
@@ -0,0 +1,92 @@
+package controllers.usermanagement.validator;
+
+import org.junit.*;
+import org.sunbird.common.exception.ProjectCommonException;
+import org.sunbird.common.models.util.JsonKey;
+import org.sunbird.common.request.Request;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class ShadowUserMigrateReqValidatorTest {
+
+    private Request request;
+    private ShadowUserMigrateReqValidator shadowUserMigrateReqValidator;
+
+
+    @Before
+    public void setUp() throws Exception {
+        request=new Request();
+
+    }
+
+    @After
+    public void tearDown() throws Exception {
+        request=null;
+    }
+
+    @Test(expected= ProjectCommonException.class)
```

Comment:
this will allow user to change status code and error message , ideally if some one is changing that then test case should fail. so do assertion for status code and error code as well.

Reply:
done